edgexfoundry/device-modbus-go
# refactor!: use latest SDK for flattened config stem

BREAKING CHANGE: location of service configuration in Consul changed

Also required updated for v3 modules

Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-modbus-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. Run EdgeX non-secure stack so Consul is running
2. Build the service locally `make build`
3. `cd cmd`
4. `./device-modbus -cp`
5. verify that the service configuration is at `http://localhost:8500/ui/dc1/kv/edgex/v3/`

## New Dependency Instructions (If applicable)
